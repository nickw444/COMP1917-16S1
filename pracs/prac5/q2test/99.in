99
10 -3 4 0 9 1 -6 6 -7 4 -9 -9 5 3 -7 -2 -6 8 -1 3 -4 0 -2 -7 -6 -3 -2 0 -5 0 4 -3 0 -5 9 -7 -10 -1 7 10 2 -1 8 2 9 -2 9 -7 1 4 -5 -8 -3 -8 9 6 -6 -5 -6 -3 2 -1 0 7 7 7 -8 4 6 5 2 -6 -4 -5 1 -9 9 -5 -2 3 0 -3 7 2 3 10 3 -10 9 8 -7 -7 10 -1 -4 -6 -4 9 7
