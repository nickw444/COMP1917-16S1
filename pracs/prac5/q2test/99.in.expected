average of array: 0.131313
